src/main/java/sggw/wzim/czasnawypad/db/dto/CreateAttractionRatingDTO.java

```diff
@@ -0,0 +1,21 @@
+package sggw.wzim.czasnawypad.db.dto;
+
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+import java.time.LocalDate;
+import java.util.Date;
```

Comment:
Don't use `java.util.Date` as it's deprecated. https://www.baeldung.com/java-convert-localdate-sql-date